app/utils.py

```diff
@@ -82,3 +84,33 @@ def fetch_cases_by_date(case_date_from, case_date_to):
                     'signed_date': contract['ActivatedDate'],
                 })
     return result
+
+
+def generate_presto_query(event_id=None, from_date=None, to_date=None):
+    date_format = "%Y-%m-%d"
```

Comment:
I think you can put `date_format` as a constant at module level. This will help you in two things:
- Instead of define this in your tests: https://github.com/evbeda/upfronts/pull/37/files#diff-713fae2c70e8430732300129a7e54cbcR768 so you can `use` this constant in your tests.
- If you share a `date_format` between methods in this file, you are going to have this module level constant.

Reply:
I realize you have `date_format` here, so it will a plus if you add this constants in your `constants`/`__init__` file

Reply:
Oh, that's right. Thanks!